app/models/manageiq/providers/hawkular/middleware_manager/event_catcher/runner.rb

```diff
@@ -0,0 +1,4 @@
+class ManageIQ::Providers::Hawkular::MiddlewareManager::EventCatcher::Runner <
+  ManageIQ::Providers::BaseManager::EventCatcher::Runner
+  include ManageIQ::Providers::Hawkular::MiddlewareManager::EventCatcherMixin
```

Comment:
Ah, gotcha.  If it's helping with testing then leave it with the mixin.  It's not really hurting anything.